packages/myst-cli/src/transforms/outputs.ts

```diff
@@ -37,7 +37,7 @@ export async function transformOutputsToCache(
 ) {
   const outputs = selectAll('output', mdast) as GenericNode[];
   // This happens sooner for notebooks
-  if (!outputs.length || kind !== SourceFileKind.Article) return;
```

Comment:
My current thinking is that we load the notebook as-is, and minify later during transformations in a single step. Is this going to have perf ramifications?

Reply:
Yeah, I think that makes a lot of sense to minify later in the transforms. We need to do it before the image/math stuff, so likely `read -> parse -> optional execution -> minify -> other transforms`.